src/pages/getinvolved.jsx

```diff
@@ -13,71 +23,187 @@ import hackathon from '../../public/images/hackathon.png'
+                <ExpandMoreIcon
+                  sx={{
+                    fontSize: '5vw',
+                    color: '#ffffff',
+                    transform: 'translateY(-3.5vw)',
+                  }}
+                />
+              </ScrollDownButton>
+            </ScrollElement>
+            <PageDesc>
+              {`There are many ways to help in the recovering of marine life,
+              especially for the Souther Resident Killer Whales that call the
+              Salish Sea home. Check out the ways you can help below!`}
+            </PageDesc>
+          </TitleScreen>
+        </Slide>
+      </TopScreen>
 
+      <Box
+        id="invite-links"
+        sx={{
+          padding: '20px',
+          display: 'flex',
+          flexDirection: 'column',
+          alignItems: 'left',
+        }}
+      >
+        {InviteLinks.map((link, i) => (
+          <ScrollLink key={i} to={link.id} smooth={true} spy={true}>
+            {link.name}
+          </ScrollLink>
+        ))}
+      </Box>
+
+      <div id="volunteer">
         <h2 className={getinvolvedStyles.particiation}>
           <strong>Volunteer</strong>
         </h2>
-
         <div>
           <div className={getinvolvedStyles.paris}>
             <div className={getinvolvedStyles.rows}>
               <h2>Citizen Scientist</h2>
               <p>
-                First and foremost, you can volunter as a citizen scientist. Not
+                {`First and foremost, you can volunter as a citizen scientist. Not
                 only an you listen for whales to marvel at the symphony of
-                ssounds they make.<br></br> but also you can listen 'for'
+                ssounds they make.`}
+                <br></br>
+                {`but also you can listen 'for'
                 whales-- helping monitor their habitat and notifying the network
-                when you hear them or a noise that could endager them.
+                when you hear them or a noise that could endager them.`}
               </p>
             </div>
           </div>
 
           <div className={getinvolvedStyles.rows}>
             <h2>In Person</h2>
             <p>
-              You can also volunteer in-person with any of them current
+              {`You can also volunteer in-person with any of them current
               Orcasound network members, or with a new organizationin your
-              neighborhood that you convince to become a new member<br></br>
+              neighborhood that you convince to become a new member`}
               <br></br>
-              Volunteer opportunities can include helping deploy or fix
+              {`Volunteer opportunities can include helping deploy or fix
               hydrophones, teaching groups how to 'listen for whales,' or
-              helping create new educational or outreach projects.
+              helping create new educational or outreach projects.`}
             </p>
```

Comment:
Thank you for trying to fix lint warnings, but since this PR is just for the top section, I would put all these changes in a separate PR (and everything below this too, Github won't let me select more than this). Otherwise it's confusing why half the page got changed in order to add the top section, also it makes it harder to review. Since you want to work on #63, you could just move these changes into a new PR

Reply:
@paulcretu I've done the requested changes. Let me know if you still find something wrong or better, I will be happy to consider it. Also, I have made a PR to fix all lint warnings and it will fix the warnings generated by this PR, so it would be better if you merge this PR first and that one later.